pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `definitions`               | Validation    | 04                 | 2019-09 | Replaced by` $defs`                                                                          |
+| `default`                   | Validation    | 01                 | No      | No                                                                                           |
+| `dependencies`              | Validation    | 03                 | 2019-09 | Replaced `requires` and replaced by `dependentSchemas` and `dependentRequired`               |
+| `dependentRequired`         | Validation    | 2019-09            | No      | Partially replaced `dependencies`                                                            |
+| `dependentSchemas`          | Core          | 2019-09            | No      | Partially replaced `dependencies`                                                            |
+| `deprecated`                | Validation    | 2019-09            | No      | No                                                                                           |
+| `disallow`                  | Core          | 01                 | 04      | Replaced by `not`                                                                            |
+| `divisibleBy`               | Validation    | 02                 | No      | Replaced by `multipleOf`                                                                     |
+| `else`                      | Core          | 07                 | No      | No                                                                                           |
+| `enum`                      | Validation    | 01                 | No      | No                                                                                           |
+| `examples`                  | Validation    | 06                 | No      | No                                                                                           |
+| `exclusiveMaximum` (legacy) | Validation    | 03                 | 06      | Replaced `maximumCanEqual`                                                                   |
+| `exclusiveMaximum`          | Validation    | 06                 | No      | Replaced `exclusiveMaximum` (legacy)                                                         |
+| `exclusiveMinimum` (legacy) | Validation    | 03                 | 06      | Replaced `minimumCanEqual`                                                                   |
+| `exclusiveMinimum`          | Validation    | 06                 | No      | Replaced `exclusiveMinimum` (legacy)                                                         |
+| `extends`                   | Core          | 01                 | 04      | Replaced by `allOf`                                                                          |
+| `format`                    | Validation    | 01                 | No      | -                                                                                            |
```

Comment:
This keyword was turned into an annotation keyword in 2019-09. Also there were many changes in the set of permitted values this keyword could take over varios versions of the specifications. Not sure if we want to document those here